styleguide/source/_patterns/03-organisms/topic-membership.md

```diff
@@ -0,0 +1,31 @@
+---
+el: ".ama__topic_membership"
+title: "Topic Membership"
+---
+
+The topic membership pattern includes a heading, a description, and a CTA button (optional).
+
+[EWL-3817](https://issues.ama-assn.org/browse/EWL-3817)
+[EWL-4214](https://issues.ama-assn.org/browse/EWL-4214)
+
+### Variant options
+* [With CTA button](./?p=organisms-topic-membership-with-cta)
+
+### Variables
+~~~
+{
+  "heading": {
+    "level": int/required
+    "text": string/required
+    "class": "ama__h2"
+  },
+  "link": string/required
+  "paragraph" : {
+    "text": string/required
+  },
+  (optional) "button": {
+    "text": string/optional
+    "style": "primary"
+  }
+}
+~~~
```

Comment:
This is a really thorough and well thought out markdown file. I appreciate the attention to detail here. Is there any way to make a version of this appear on the "with CTA" variant when you click "Pattern info" in PatternLab?